v2/common/src/main/java/com/google/cloud/teleport/v2/options/CommonTemplateOptions.java

```diff
@@ -49,4 +51,23 @@ public interface CommonTemplateOptions extends PipelineOptions {
   String getExtraFilesToStage();
 
   void setExtraFilesToStage(String extraFilesToStage);
+
+  @TemplateParameter.Enum(
+      order = 33,
+      optional = true,
+      enumOptions = {
+        @TemplateParameter.TemplateEnumOption("OFF"),
+        @TemplateParameter.TemplateEnumOption("ERROR"),
+        @TemplateParameter.TemplateEnumOption("WARN"),
+        @TemplateParameter.TemplateEnumOption("INFO"),
+        @TemplateParameter.TemplateEnumOption("TRACE"),
+        @TemplateParameter.TemplateEnumOption("DEBUG")
+      },
+      description = "Log level in the workers, defaults to INFO",
```

Comment:
What is the current default ? Let's ensure that we retain that.

Reply:
Default is info only.